CMakeLists.txt

```diff
@@ -373,6 +373,8 @@ include(nuttx_add_application)
 include(nuttx_add_romfs)
 include(nuttx_add_symtab)
 include(nuttx_add_module)
+include(nuttx_add_dependencies)
+include(nuttx_apps_headers)
```

Comment:
nuttx_export_header

Reply:
> change to nuttx_export_header

this has been changed in patchset2.
do you mean squash these two commits? do we need to keep the discussion and revison?

Reply:
> > change to nuttx_export_header
> 
> this has been changed in patchset2. do you mean squash these two commits? do we need to keep the discussion and revison?

Done 🙏